configstack/running_limit.go

```diff
@@ -7,15 +7,20 @@ import (
 	"github.com/fatih/color"
 )
 
+const waitTimeBetweenThread = 500
```

Comment:
You're using the wording "thread" here and I feel that it may be a little misleading. It might give the impression that we're dealing with threads under the hood when it's not really the case.

Perhaps you could rename this to something along the lines of `slowStartDelay` or maybe `progressiveStartDelay`.

Reply:
I had just copy pasted the old code, I didn't even pay attention to the name 😅